src/Greylist.js

```diff
@@ -0,0 +1,27 @@
+const Validator = require('./Validator');
+const util = require('./utility');
+const semesterAccount = require('../config').semesterAccount;
+
+// Closes the PR if it fulfills the title check pattern
+// Bad title PR from students will fall through
+module.exports = (accuser, account, repository) => {
+  let validator = new Validator(accuser, account, repository);
+
+  let filterBlock = (repo, issue) => {
+    return issue.pull_request;
+  };
+
+  let doBlock = (repo, issue) => {
+    const titlePattern = util._titleRegex;
```

Comment:
Can make the regex a parameter, for future flexibility?

Reply:
hmmm. i think it won't help much because if look at each or the `xxxlist.js`, we are doing something very specific with the regex result. if we allow the regex to be passed in, it can end up in a giant `if-else` statement in the class. I rather restrict it so that separate functionality must be implemented in separate files

Reply:
okay sure